DRAFT: Multi-Party Payout.md

```diff
@@ -0,0 +1,204 @@
+<pre>
+  DIP: 000x
+  Title: Multi-Party Payouts
+  Author(s): Timothy Munsell,
+  Comments-Summary: No comments yet.
+  Status: Draft
+  Type: Standard
+  Created: 2021-05-08
+  License: MIT License
+</pre>
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Abstract](#abstract)
+- [Motivation and Previous System](#motivation-and-previous-system)
+  - [Problems with the previous system](#problems-with-the-previous-system)
+- [Prior Work](#prior-work)
+- [New Stake Holder Reward Fields within ProRegTx and ProUpRegTx](#new-stake-holder-reward-fields-within-proregtx-and-proupregtx)
+- [Registering a Masternode (ProRegTx)](#registering-a-masternode-proregtx)
+- [Updating Masternode Information](#updating-masternode-information)
+  - [Updating Registrar of Masternode (ProUpRegTx)](#updating-registrar-of-masternode-proupregtx)
+- [Validation Rules](#validation-rules)
+  - [ProRegTx and ProUpRegTx](#proregtx-and-proupregtx)
+  - [ProUpRevTx](#prouprevtx)
+- [Masternode Rewards](#masternode-rewards)
+- [Copyright](#copyright)
+
+## Abstract
+
+This DIP builds on the chain consensus for masternode lists laid forth in DIP0003 and provides for multi-party payout beyond the single owner + operator framework of the current system.
+
+## Motivation and Previous System
+
+In the previous system, Masternodes gained entry to the masternode list after the owner created a ProRegTx (DIP3). This tx provided key IDs for up-to 2 roles that would receive masternode rewards payouts:

```

Comment:
As "Previous System" is in the title, we could take that out of the description. 
Perhaps something along the lines of:
```
Masternodes gained entry to the masternode list in the previous system after the owner created a ProRegTx (DIP3). This tx provided key IDs for up to 2 roles that would receive masternode rewards payouts:
```